Source/carthage/Bootstrap.swift

```diff
@@ -26,7 +26,20 @@ public struct BootstrapCommand: CommandType {
 					carthage.println(formatting.bullets + "No Cartfile.resolved found, updating dependencies")
 					return project.updateDependencies(shouldCheckout: options.checkoutAfterUpdate)
 				}
-
+				
+				if let depsToUpdate = options.dependenciesToUpdate {
+					let resolvedCartfileContents = try! String(contentsOfURL: project.resolvedCartfileURL, encoding: NSUTF8StringEncoding)
+					let resolvedCartfile = ResolvedCartfile.fromString(resolvedCartfileContents).value!
+					
+					let resolvedDependencyNames = Set<String>(resolvedCartfile.dependencies.map { $0.project.name })
+					let updatedDependencyNames = Set<String>(depsToUpdate)
+					let unresolvedDependencyNames = updatedDependencyNames.subtract(resolvedDependencyNames)
```

Comment:
This can be simplified a bit further because:

1. You shouldn't need to specify the value type for the set
2. The parameter to `Set.subtract' doesn't need to be a set
3. `updatedDependencyNames` doesn't need to be its own variable

Something like this should suffice:

```swift
let resolvedDependencyNames = resolvedCartfile.dependencies.map { $0.project.name }
if !Set(depsToUpdate).subtract(resolvedDependencyNames).isEmpty {
   …
}
```